localdev/management/commands/populate_local_starters.py

```diff
@@ -0,0 +1,67 @@
+""" Scans the starter project directory in the local filesystem and creates/updates relevant db records """
+import os
+from collections import defaultdict
+
+from django.conf import settings
+from django.core.management import BaseCommand
+
+from websites.constants import WEBSITE_CONFIG_FILENAME, STARTER_SOURCE_LOCAL
+from websites.models import WebsiteStarter
+
+LOCAL_STARTERS_DIR = "localdev/starters/"
+
+
+class Command(BaseCommand):
```

Comment:
Correct me if I'm wrong because I'm not super familiar with our Python development practices, but should we have some unit tests for this file that try and initialize the example config and read some properties of it?  `backpopulate_ocw_courses.py` doesn't have any tests either, maybe it's not necessary?

Reply:
We rarely add unit tests to management commands. That has bitten us once or twice but in most cases I don't see the benefit outweighing the added work

Reply:
That makes sense